File: Assemblies/Protean.Tools/Integration/Twitter/TwitterVB2/Objects/ATwiCliAudioFile.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Protean.Tools.Integration.Twitter.TwitterVB2
{
    public partial class ATwiCliAudioFile : ATwiCliFile
    {
        #region TCAF Constants
        public const string TCAF_AUDIO_STRING = "audio.mp3";
        public const string TCAF_AUDIO_EXTENSION = ".mp3";
        #endregion

        public override string GetDirectURL()
        {
            return TCF_DIRECT_URL + TCAF_AUDIO_STRING + "?id=" + this.ID;
        }

        public override string GetExtension()
        {
            return TCAF_AUDIO_EXTENSION;


        }
    }
}

File: Assemblies/Protean.Tools.vb Deprecated/Integration/Twitter/TwitterVB2/Objects/ATwiCliAudioFile.vb
Namespace Integration.Twitter.TwitterVB2
    Public Class ATwiCliAudioFile
        Inherits ATwiCliFile
#Region "TCAF Constants"
        Public Const TCAF_AUDIO_STRING As String = "audio.mp3"
        Public Const TCAF_AUDIO_EXTENSION As String = ".mp3"
#End Region

        Public Overrides Function GetDirectURL() As String
            Return TCF_DIRECT_URL & TCAF_AUDIO_STRING & "?id=" & Me.ID
        End Function

        Public Overrides Function GetExtension() As String
            Return TCAF_AUDIO_EXTENSION


        End Function
    End Class


End Namespace